src/common/components/share/styles.css

```diff
@@ -18,8 +18,12 @@
 }
 
 .clipboard {
-  background-image: url(https://assets.ubuntu.com/v1/9cbcbbc7-copy-clipboard.svg);
-  background-size: 12px;
+  background: transparent;
```

Comment:
@richmccartney @bartaz share icons were a topic of conversation in our Blog theme inventory workshop last week. The plan is to generate a set of social share icons which is in our backlog of work to be done.